CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Releases
 
+## Version 0.14.0
+
+Date: 2022-09-30
+
+This release focuses on three main themes, support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript), quality of life improvements to make it easier to build responsive and performant applications and finally improvements to static typing and docstrings to enhance the developer experience.
+
+Many, many thanks to everyone who filed issues or contributed to this release. In particular we would like to thank @janimo, @xavArtley, @thuydotm, @jmosbacher, @dmarx, 2WoLpH, @ipopa144, @sdc50 and the core team consisting of @philippjfr, @Hoxbro, @maximlt and @MarcSkovMadsen.
+
+### Features
+
+- Add support for converting Panel apps to pyscript/pyodide (#3817, #3830, #3851, #3856, #3857, #3858, #3860, #3861, #3863, #3864, #3868, #3878)
+- Manage our own CDN (#3867, #3870)
+- Add `pn.widget` helper function (#1826)
+- Add ability to `defer_load` of components (#3882)
+- Add `config.exception_handler` to easily capture, log and notify users about errors (#3893)
+- Implement `pn.cache` function for memoization support (#2411)
+- Rewrite server extension to run Panel applications in kernels (#3763)
```

Comment:
```suggestion
- Rewrite server extension to run Panel applications in kernels so that previews run in the same environment as the deployed app (#3763)
```

Reply:
so that previews run in the same environment as the notebook and code....